packages/workspace/src/browser/workspace-preferences.ts

```diff
@@ -14,7 +14,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { interfaces } from "inversify";
+import { interfaces } from 'inversify';
```

Comment:
please don't introduce unrelated changes, a PR is big enough already, we also don't have the rule to force using of single quote vs double, not sure why you are reformatting them

Reply:
i am ok with double quotes. i change them to single quotes when i find doubles and singles are mixed - just want the code to have a more consistent look .